src/scripts/main.js

```diff
@@ -354,7 +354,36 @@
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+function createRow(person) {
+  const row = document.createElement('tr');
+
+  const nameCell = document.createElement('td');
+  const sexCell = document.createElement('td');
+  const bornCell = document.createElement('td');
+  const diedCell = document.createElement('td');
+  const ageCell = document.createElement('td');
+  const centuryCell = document.createElement('td');
+
+  nameCell.textContent = person.name;
+  sexCell.textContent = person.sex;
+  bornCell.textContent = person.born;
+  diedCell.textContent = person.died;
+  ageCell.textContent = person.died - person.born;
+  centuryCell.textContent = Math.ceil(person.died / 100 - 1);
+
+  row.appendChild(nameCell);
+  row.appendChild(sexCell);
+  row.appendChild(bornCell);
+  row.appendChild(diedCell);
+  row.appendChild(ageCell);
+  row.appendChild(centuryCell);
+
+  return row;
+}
+
+people.forEach(person => {
+  const row = createRow(person);
+  table.appendChild(row);
+});
```

Comment:
Performance Issue: You're appending each row to the table inside the loop. This causes the browser to reflow the page multiple times, which can be expensive in terms of performance. A better approach would be to create a document fragment, append the rows to the fragment, and then append the fragment to the table after the loop. This way, the browser only reflows the page once.